### PR TITLE
Fix tests 4.1.3, 4.1.4, 4.2.1, 4.2.2, 4.2.3, 4.2.4 and way to process data in woketo

### LINKED ISSUE #27: Solve wstest 4.1.3
See #1

### PR DESCRIPTION
**Pull request by Nek-**
Please notice that the new way of processing data is not great in term of maintainability. This is an echo to #23 .

This PR should close #27

image
image

*Please notice it also fixes some tests in 9.x.*

### CODE REVIEW

File: src/Rfc6455/Message.php [hunk 1]
```diff
@@ -40,17 +42,40 @@ public function __construct()
...
+//
+//            if (!$this->isComplete() && !empty($data)) {
+//                $data = $this->addData($data);
+//            }
+//        } catch (IncompleteFrameException $e) {
+//            $this->buffer .= $data;
+//            return '';
+//        }
+//
+//        return $data;
```
Comment (folliked) [inline src/Rfc6455/Message.php]: no code in comments please

File: src/Exception/Frame/TooBigFrameException.php [hunk 2]
```diff
@@ -10,9 +10,21 @@
...
 namespace Nekland\Woketo\Exception\Frame;
 
+use Exception;
 use Nekland\Woketo\Exception\LimitationException;
 
 class TooBigFrameException extends LimitationException
 {
+    private $maxLength;
 
+    public function __construct($maxLength, $message = 'The frame is too big to be processed.')
```
Comment (folliked) [inline src/Exception/Frame/TooBigFrameException.php]: no type ?

File: src/Rfc6455/Frame.php [hunk 3]
```diff
@@ -8,6 +8,7 @@
  * on the root directory of this project
  */
 
+declare(strict_types=1);
```
Comment (folliked) [inline src/Rfc6455/Frame.php]: why ? no good for performance ...

Comment (Nek-) [inline src/Rfc6455/Frame.php]: Because sometimes it's really important. (and it is for some parts of woketo)

Comment (folliked) [inline src/Rfc6455/Frame.php]: I understand but it is in a central class,  so it should be interesting a load tests for the impact. But ok

**Pull request merged**